.github/workflows/comptime_ci.yml

```diff
@@ -0,0 +1,51 @@
+name: Comptime CI
```

Comment:
This file can be eliminated completely. See the comment for `use_flag_comptime_values.vv`.